tools/bookrunner/src/util.rs

```diff
@@ -113,7 +113,7 @@ fn try_parse_args(cur_args: Vec<String>, name: &str, line: &str) -> Vec<String>
     let name = format!("{}-flags:", name);
     let mut split = line.split(&name).skip(1);
     let args: Vec<String> = if let Some(rest) = split.next() {
-        rest.trim().split_whitespace().map(String::from).collect()
+        rest.split_whitespace().map(String::from).collect()
```

Comment:
Does this change the semantics?

Reply:
Nope! That's apparently why clippy has this lint. :)